### DISCUSSION TITLE
Missing Device FGBS-222 //  Fibaro Smart Implants

### DISCUSSION THREAD

**Discussion by OttarNOR** [zwave-js/zwave-js#5494] [General] [answered]
Hi,
I use tow Fibaro Smart Implants in my smart house, for controlling my garage door and monitoring my doorbell. 
While I'm migrating my smart home to from Fibaro to HA, I have discovered that this device is missing from the Z-Wave JS Config.

**Is it possible to add the FGBS-222 device, with all its parameter?**
https://manuals.fibaro.com/smart-implant/

**Comment by AlCalzone** [comment]
We do have that: https://devices.zwave-js.io/?jumpTo=0x010f:0x0502:0x1000:0.0
Maybe yours is using a different device ID?

**Comment by OttarNOR** [comment, marked as answer]
Thank you for getting in touch,Great that you have this device. Do you have any suggestions on how I can get it working? 
I can't access the parameters
 
![B43866E7-96B5-4DC4-B61B-9ACE64CD3D0A](https://user-images.githubusercontent.com/87257784/221610219-eca32dcf-2f6a-4471-af02-d437c8c5092f.jpeg)
![9C8A9FFF-EDAA-494D-8660-50163AEAB290](https://user-images.githubusercontent.com/87257784/221610225-5bf0303d-4c7d-4827-9349-931858430d0d.jpeg)

**Reply by AlCalzone** [reply]
Please share the node diagnostics

**Reply by OttarNOR** [reply]
Hi @AlCalzone ,
Sorry for my late response, been away on winter vacation.
See the diagnostics file, linked below. Your help is much appreciated. 

[zwave_js-4b81523e6bf6ba5f9bca44336e527b30-Node 15-73be416747dbc624574315444a55e410.json.txt](https://github.com/zwave-js/node-zwave-js/files/10892167/zwave_js-4b81523e6bf6ba5f9bca44336e527b30-Node.15-73be416747dbc624574315444a55e410.json.txt)

**Reply by AlCalzone** [reply]
I think the issue is that the device was originally included into the network using Security, but you didn't set up those security keys in HA.

**Reply by OttarNOR** [reply]
Hi @AlCalzone,
**Quick update**.  Your prediction was correct.
**Error**: I had included the node using the mobile app and scanning the QR code. Secure inclusion was then not promoted.
**Fix**: I excluded the node and manually included it in secure mode.

The node/senior is now included as it should with all parameters available.
